src/server.py

```diff
@@ -17,7 +17,7 @@
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.openapi.utils import get_openapi
 
-AC_VERSION = '3.0.0'
+AC_VERSION = '2.9.10'
```

Comment:
Did this version reversion happen on purpose?  This is a pretty big change to the AC API so I think it's fair to give it a major version bump.

Reply:
sure thing i can do that i was aligning it with the docker image releases, will bump this to 3.0.0 and create a new release then